db/seeds.rb

```diff
@@ -0,0 +1,13 @@
+User.create!(
+  name: 'Deepak Mahakale',
+  email: 'deepak@weboapps.com',
+  api_key: 'abcdefgh12345678abcdefgh'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.